modules/querier/stats/stats.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package stats;
+
+option go_package = "github.com/grafana/tempo/modules/querier/stats";
```

Comment:
I think maybe it has been a while since the protos had been generated, but more recent versions of protoc require that the go_package name contain more than just the string, so this has been updated here to reflect the name of the package.  Same goes for the two frontend proto packages.